presto-orc/src/test/java/com/facebook/presto/orc/TestingHiveOrcLocalMemoryContext.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import com.facebook.presto.memory.context.LocalMemoryContext;
+import com.google.common.util.concurrent.ListenableFuture;
+
+import static java.util.Objects.requireNonNull;
+
+public class TestingHiveOrcLocalMemoryContext
+        implements OrcLocalMemoryContext
+{
+    /*
```

Comment:
javadoc (staring with /**) is usually place right before the class definition without a line break. e.g.
```java
/**
 * This class is a copy of HiveOrcLocalMemoryContext.
 * Any changes made to HiveOrcLocalMemoryContext should be made here as well
 */